lib/openactive/validators/array_of_validator.rb

```diff
@@ -0,0 +1,72 @@
+module OpenActive
+  module Validators
+    class ArrayOfValidator < BaseValidator
+      attr_accessor :item_validator
+
+      def initialize(item_validator)
+        @item_validator = item_validator
+      end
+
+      # Coerce given value to the type the validator is validating against.
+      # PLEASE NOTE: no checks are performed on the given $value.
+      # It is therefore recommended to call the "run" method first before this.
+      #
+      # @param value mixed The value to coerce.
+      # @return mixed The same value.
+
+      def coerce(value)
+        # NOTE: OpenActive is more strict than schema.org in this regard, so commenting out this for now
+        # $nullValidator = NullValidator.new()
+
+        # If we are providing a single item of the item_validator type (or null)
+        # Put the value inside an array
+        # if(
+        #     $nullValidator.run($value) === true ||
+        #     self.item_validator.run($value) === true
+        # ) {
+        #     return [$value]
+        # }
+
+        # Otherwise this is a no-op
+        # return [value] unless  value.is_a?(Array)
```

Comment:
Why commented out "for now"? Also this is PHP code that is commented out.

Reply:
Removed + re-worded comments